docs/serving/load-balancing/target-burst-capacity.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Configuring target burst capacity"
+linkTitle: "Configuring target burst capacity"
+weight: 50
+type: "docs"
+aliases:
+    - /docs/serving/autoscaling/target-burst-capacity
+---
+
+_Target burst capacity_ determines the size of traffic burst a Knative application can handle without buffering.
+If a traffic burst is too large for an application to handle without buffering, the activator will be placed in the request path to protect the revision and optimize request load balancing.
+The activator can also quickly spin up additional pods for capacity, and throttle how quickly requests are sent to pods.
```

Comment:
It is not clear what pods can be spun up. If revision — then autoscaler does that, if activator pods then Kubernets HPA does that.